### PR TITLE
How to create git book

### PR DESCRIPTION
**Pull request by amitaibu**
#16

https://www.gitbook.com/book/amitaibu/the-gizra-way/edit#/edit/16-create-gitBook/how_and_what_to_document.md

### CODE REVIEW

**Comment by amitaibu** [comment]
@RachelBaram ready for review

File: how_and_what_to_document.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+GitBook's repository is being automatically synced to GitHub, so you may change it in one of them, and it is guaranteed to be reflected in the other.
+
+As a privileged user, start by [editing](https://www.gitbook.com/book/amitaibu/the-gizra-way/details) the book
+
+
+
+![](images/gitbook/image1.jpg)
+
+Create a new branch, you will work on a side branch, which will not immediately be reflected on the official book. This allows you to go wild with your edits, without the fear of breaking the live site.
+
```
Comment (RachelBaram) [inline how_and_what_to_document.md:12]: Is this the correct place to mention how to name the brance? WIP in front until a pull request is created.

Comment (amitaibu) [inline how_and_what_to_document.md:12]: > Is this the correct place to mention how to name the brance?

We don't really care about the branch name.

> WIP in front until a pull request is created.

The WIP part is in the GitHub chapter that is linked below

File: how_and_what_to_document.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+GitBook's repository is being automatically synced to GitHub, so you may change it in one of them, and it is guaranteed to be reflected in the other.
+
+As a privileged user, start by [editing](https://www.gitbook.com/book/amitaibu/the-gizra-way/details) the book
+
+
+
+![](images/gitbook/image1.jpg)
+
+Create a new branch, you will work on a side branch, which will not immediately be reflected on the official book. This allows you to go wild with your edits, without the fear of breaking the live site.
```
Comment (RachelBaram) [inline how_and_what_to_document.md]: Not sure I edited this correctly about the new and side brance (which are the same).

**Comment by RachelBaram** [comment]
Review and edits done.

**Comment by amitaibu** [comment]
Thanks.

**Pull request merged**